src/PackagePublisher/Program.cs

```diff
@@ -20,11 +20,12 @@ public static class Program
         private const string ApiKeyEnvironmentVariableName = "APIKey";
         private static CommandOption _packagesDirectory;
         private static CommandOption _feedToUploadTo;
-        private static CommandOption _maxRetryCountOption;
 
         // Do NOT log the key
         private static string _apiKey = null;
-        private static int _maxRetryCount = 10;
+        private static int _maxRetryCount = 5;
```

Comment:
Change to `private const _maxRetryCount` since the command line param has been removed.